client/QueryCompletionInformation/QueryCompletionInformationExample/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Kusto.Cloud.Platform.Data;
+using Kusto.Data;
+using Kusto.Data.Data;
+using Kusto.Data.Net.Client;
+using System.Data;
+using System.Linq;
+
+namespace QueryCompletionInformationExample
+{
+    /// <summary>
+    /// This example demonstrates how to retrieve the QueryCompletionInformation from a Kusto query response stream.
```

Comment:
Maybe explain here what this table is or refer to docs. User might not understand that its always part of the returned response